torch/_inductor/ir.py

```diff
@@ -2931,22 +2934,6 @@ class TritonTemplateBuffer(TemplateBuffer):
     pass
 
 
-class CUDATemplateBuffer(TemplateBuffer):
```

Comment:
Why this change? It's part of IR so seems that it's better to keep it here.

Reply:
In order to prevent circular imports, I pushed logic into the CUDATemplateBuffer that requires imports which we don't want to place at the top of ir.py

Reply:
This is already the specific one.

Reply:
Moved this back to ir.py after some changes which allowed to do so.